app/controllers/accounts_controller.rb

```diff
@@ -42,7 +42,7 @@ def show
       format.json do
         skip_session!
 
-        render_cached_json(['activitypub', 'actor', @account.cache_key], content_type: 'application/activity+json') do
+        render_cached_json(['activitypub', 'actor', @account], content_type: 'application/activity+json') do
```

Comment:
If an array is given to the key of Rails.cache.fetch, the key will be like `activitypub/actor/accounts/1`.